restore.cmd

```diff
@@ -1,3 +1,3 @@
 @echo off
-powershell -NoLogo -NoProfile -ExecutionPolicy ByPass -Command "& """%~dp0eng\common\build.ps1""" -restore %*"
+powershell -ExecutionPolicy ByPass -NoProfile -command "& """%~dp0eng\common\Build.ps1""" -restore -msbuildEngine dotnet %*"
```

Comment:
`-restore -msbuildEngine dotnet %*"` are pulled from dotnet-format repo. 
These are added back By Jon https://github.com/dotnet/format/commit/3c12e26cb5508a5793087eb4c868cfe3c9908e1e

Not sure if it's fit the sdk

Reply:
Should we add back the `-NoLogo`?